recipes/ctre/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1)
```

```suggestion
cmake_minimum_required(VERSION 2.8.11)
```
You are not using cmake 3+ features here, and they are not required anyway.